deployment/frontend/src/server/api/routers/dataset.ts

```diff
@@ -1052,10 +1052,10 @@ export const DatasetRouter = createTRPCRouter({
                             'Error in sending approval status notification'
                         )
                     }
-                    const resource = data.result.package_data.resources.find(
+                    const _resource = data.result.package_data.resources.find(
```

Comment:
Take a look at this function and where it uses "resource" above this part of the code.